examples/example5.cpp

```diff
@@ -2,6 +2,16 @@
 
 #include <backend_iface.hpp>
 
+void print_dpnp_array(double* arr, size_t size)
```

Comment:
This function is better to use from
https://github.com/IntelPython/dpnp/blob/6d2aaafe24ac927749dcd3c729d798988ffadb98/dpnp/backend/backend_utils.hpp#L52